class-tgm-plugin-activation.php

```diff
@@ -439,7 +439,7 @@ public function install_plugins_page() {
 				<?php $plugin_table->prepare_items(); ?>
 
 				<?php
-				if ( isset( $this->message ) ) {
+				if ( isset( $this->message ) && is_string( $this->message ) && '' !== $this->message ) {
```

Comment:
Interesting... I always ask them to cut the actual quiz out if they video it - which one did you see ? There are actually some five different versions of the quiz (different sets of questions).